src/lib/deploy.js

```diff
@@ -180,50 +180,70 @@ async function deploy({ network, yes }) {
   let zkAppPrivateKey = PrivateKey.fromBase58(privateKey); //  The private key of the zkApp
   let zkAppAddress = zkAppPrivateKey.toPublicKey(); //  The public key of the zkApp
 
-  let verificationKey = await step(
+  const verificationKey = await step(
```

Comment:
Is there a reason you are using let instead of const on this line?
 let { verificationKey } = await zkApp.compile(zkAppAddress);

Reply:
Nope, no particular reason, it was leftover from some testing in a previous PR.